### ISSUE TITLE
dct:issued validation on opendata.swiss: don't publish datasets that are not yet released

### ISSUE THREAD

**Issue by sabinem** [opendata-swiss/dcat_ap_ch#171]
**Property**: 'dct:issued'
**Class**: ´dcat:Dataset'
**Implementation**: in the implementation on opendata.swiss that dataset can be set to published despite the fact that the release date is in the future. This should not be possible
**Conformance**: The usage note in DCAT-AP CH states that is the the release date is in the future the dataset will not show as a published dataset.
**Proposal**: Only show datasets as published if the release date in not in the future. For "manuelle Datenerfassung" and XML Import don't allow to publish the dataset if the release date is in the future.

**Comment by Contatto60** [comment]
... Another possibility would be just to forget about this usage note. Do we really need such a mechanism? What do data publishers think about it? Do they want to be able to set a date in the future?

**Comment by AFoletti** [comment]
A date in the future does in my opinion not make much sense. Moreover, with it I am sort of making a promise I am not sure I can maintain (because delays always happen).
So no, no date in the future in my opinion. Better no value

**Comment by metaodi** [comment]
I agree with the original proposal of @sabinem to remove the usage note. This is not up to the standard to decide. It's up to opendata.swiss to add such a mechanism if needed.
